core/electron/src/backend/ElectronHost.ts

```diff
@@ -123,8 +123,8 @@ export class ElectronHost {
     // However, if the file doesn't exist, it's fine to return an invalid path here - the request will just fail with net::ERR_FILE_NOT_FOUND
     try {
       assetPath = fs.realpathSync(assetPath);
-    } catch (error) {
-      // eslint-disable-next-line no-console
+    } catch {
+      // // eslint-disable-next-line no-console

```

Comment:
double `//`?

Reply:
```suggestion
      // eslint-disable-next-line no-console

```

Reply:
I did that in a number of places - the actual console.log below is commented out - my preference would be to remove _both_ commented-out lines, but thought that was overstepping in a lint PR.

I saw this a lot in tests and figured if these are there for quick debugging it's not hard to uncomment both lines.  I can always just remove all double-commented directives tho, that probably makes more sense if the console.logs should always be temporary.  Thoughts?

Reply:
the reason for the double-comment btw is that these are now flagged as unused directives - so the above suggestion will break